Cargo.toml

```diff
@@ -7,5 +7,8 @@ authors = ["Jenny Mankin <jenny.mankin@crowdstrike.com>",
            
 [dependencies]
 byteorder = "*"
+
+[target.'cfg(unix)'.dependencies]
```

Comment:
As per a discussion with @tkreuzer, he and other Windows developers would prefer not to pull down kvm_bindings for the use of data structures. So instead of wrapping and exporting relevant kvm_bindings data structures as generic structures, I've used conditional compilation for Unix-based and Windows platforms to remove the dependency on kvm_bindings and reproduced the relevant structures as generic structures for Windows platforms.